azure_arc_app_services_jumpstart/aks/arm_template/artifacts/deployApiMgmt.ps1

```diff
@@ -0,0 +1,83 @@
+Start-Transcript -Path C:\Temp\deployApiMgmt.log
+
+# Make sure extensions are added
+Write-Host "`n"
+Write-Host "Make sure extensions are installed"
+Write-Host "`n"
+az extension add --name k8s-extension
+az extension update --name k8s-extension
```

Comment:
We have this as part of the [_AppServicesLogonScript.ps1_](https://github.com/microsoft/azure_arc/blob/main/azure_arc_app_services_jumpstart/aks/arm_template/artifacts/AppServicesLogonScript.ps1#L57). Any specific reason to incl. this here as well?

Reply:
did not realize, removed